src/data/baseline-data.js

```diff
@@ -0,0 +1,724 @@
+	["text-wrap-style", 0],
+	["touch-action", 10],
+	["transform-box", 5],
+	["transform", 10],
+	["backface-visibility", 10],
+	["perspective", 10],
+	["perspective-origin", 10],
+	["transform-style", 10],
+	["transition-behavior", 5],
+	["transition", 10],
+	["transition-delay", 10],
+	["transition-duration", 10],
+	["transition-property", 10],
+	["transition-timing-function", 10],
+	["user-select", 0],
+	["vertical-align", 10],
+	["writing-mode", 10],
+	["view-transition-class", 0],
+	["view-transition-name", 0],
+	["visibility", 10],
+	["white-space", 10],
+	["white-space-collapse", 5],
+	["orphans", 0],
+	["widows", 0],
+	["will-change", 10],
+	["word-break", 10],
+	["z-index", 10],
+	["zoom", 5],
+]);
+export const atRules = new Map([
+	["position-try", 0],
+	["keyframes", 10],
+	["import", 10],
+	["layer", 10],
+	["charset", 10],
+	["media", 10],
+	["font-face", 10],
+	["container", 5],
+	["counter-style", 5],
+	["view-transition", 0],
+	["media.display-mode", 0],
+	["font-face.src", 10],
+	["font-palette-values", 5],
+	["font-feature-values", 5],
+	["namespace", 10],
+	["page", 0],
+	["page.size", 0],
+	["media.prefers-color-scheme", 10],
+	["property", 5],
+	["scope", 0],
+	["starting-style", 0],
+	["supports", 10],
+]);
+export const types = new Map([
+	["abs", 0],
+	["sign", 0],
+	["anchor", 0],
+	["anchor-size", 0],
+	["time", 10],
+	["attr", 10],
+	["attr.type-or-unit", 0],
+	["blend-mode", 10],
+	["line-style", 10],
+	["calc", 10],
+	["calc-constant", 5],
+	["calc-size", 0],
+	["length", 10],
+	["global_keywords", 10],
+	["color", 10],
+	["color.color", 5],
+	["gradient", 10],
+	["gradient.conic-gradient", 10],
+	["string", 10],
+	["counter", 10],
+	["counters", 10],
+	["image", 10],
+	["easing-function", 10],
+	["exp", 5],
+	["hypot", 5],
+	["log", 5],
+	["pow", 5],
+	["sqrt", 5],
+	["filter-function", 10],
+	["url", 10],
+	["gradient.linear-gradient", 10],
+	["gradient.radial-gradient", 10],
+	["gradient.repeating-conic-gradient", 5],
+	["gradient.repeating-linear-gradient", 10],
+	["gradient.repeating-radial-gradient", 10],
+	["flex", 10],
+	["color.hsl", 10],
+	["color.hwb", 10],
+	["color.lab", 5],
+	["color.lch", 5],
+	["ratio", 10],
+	["clamp", 10],
+	["max", 10],
+	["min", 10],
+	["ray", 5],
+	["color.named-color", 10],
+	["color.oklab", 5],
+	["color.oklch", 5],
+	["number", 10],
+	["overflow", 5],
+	["image.paint", 0],
+	["basic-shape", 10],
+	["basic-shape.path", 0],
+	["color.rgb", 10],
+	["resolution", 5],
+	["color.rgb_hexadecimal_notation", 10],
+	["mod", 5],
+	["rem", 5],
+	["round", 5],
+	["easing-function.steps", 10],
+	["color.system-color", 10],
+	["angle", 10],
+	["angle-percentage", 10],
+	["position", 10],
+	["transform-function", 10],
+	["transform-function.perspective", 10],
+	["acos", 5],
+	["asin", 5],
+	["atan", 5],
+	["atan2", 5],
+	["cos", 5],
+	["sin", 5],
+	["tan", 5],
+	["dimension", 10],
+	["length-percentage", 10],
+	["percentage", 10],
+	["integer", 10],
+]);
+export const selectors = new Map([
```

Comment:
For `selectors`, feature detection could be fairly straightforward. Some like `:has` would have a single `:` and show up in the AST as `PseudoClassSelector`, others like `::before` would have 2 `::` and show up in the AST as a `PseudoElementSelector`. 

I don't think there's a way to differentiate between the two classes based on this data.

There are also some items here that are not actually verbatim selectors- for instance, "class" refers to the `.` notation for selecting classes.

Reply:
Yeah, that's the challenge I ran into so similar to types, I figured I'd just make the data was there and I'd come back to it later. Probably will need to be addressed similar to types where there needs to be a separate dictionary indicating which are pseudoclasses vs. pseudoelements vs. other.